frontend/src/pages/Transactions/components/BridgeInput.tsx

```diff
@@ -84,17 +125,17 @@ const BridgeInput = ({bridgeTxFee, getCardanoTxFee, getEthTxFee, submit, loading
   const minDfmValue = chain === ChainEnum.Nexus ? 
```

Comment:
for skyline, you should have maxAmountDfm, and maxWrappedAmount
maxWrappedAmount should be balance of a wrapped token
maxAmountDfm should be calculated the same, but minDfmValue should be:
appSettings.minUtxoValue[sourceChain]

Reply:
we might actually need a new cardano-api endpoint for getting the maxAmountDfm